azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_azurerm.go

```diff
@@ -181,30 +206,41 @@ func resourceServiceEndpointAzureRMRead(d *schema.ResourceData, m interface{}) e
 		return fmt.Errorf(" looking up service endpoint given ID (%v) and project ID (%v): %v", getArgs.EndpointId, getArgs.Project, err)
 	}
 
-	flattenServiceEndpointAzureRM(d, serviceEndpoint, (*serviceEndpoint.ServiceEndpointProjectReferences)[0].ProjectReference.Id)
+	validate, ok := d.Get("validate").(bool)
```

Comment:
You can set `validation` directly in read instead of `flattenServiceEndpointAzureRM`
```suggestion
	d.Set("validate", d.Get("validate").(bool))
```